README.rst

```diff
@@ -1,7 +1,62 @@
-========
-Overview
-========
+==============
+pytest-astropy
+==============
 
-This is a meta package that pulls in dependencies on each of the new standalone
-pytest plugins that are used by the astropy core and affiliated packages. It
-is currently WIP.
+This is a meta-package that pulls in the dependencies that are used by
+`astropy`_ and `affiliated packages`_ for testing.
```

Comment:
"and many of the affiliated packages", not all use the astropy testing framework

Reply:
Some packages that use the test helpers are not even affiliated, so this sentence needs to be more inclusive.